source/tests/utils_test.h

```diff
@@ -40,6 +40,8 @@ struct RTStopTime {
     bool _departure_skipped = false;
     bool _arrival_skipped = false;
     bool _is_added = false;
+    bool _deleted_for_detour = false;
```

Comment:
minor (as it's for testing): Not fan of this as it can be both `_departure_skipped` and `_added_for_detour`

Reply:
I think once all the cases are are handled, a refactor will ne necessary.